task/import.go

```diff
@@ -97,16 +97,16 @@ func TaskImport(tctx *TaskContext) (*TaskHandlerOutput, error) {
 
 func getFile(ctx context.Context, osSess drivers.OSSession, cfg ImportTaskConfig, params api.UploadTaskParams, vodDecryptPrivateKey string) (name string, size uint64, content io.ReadCloser, err error) {
 	name, size, content, err = getFileRaw(ctx, osSess, cfg, params)
-	if err != nil || params.Encryption.EncryptedKey == "" {
+	if err != nil || params.Encryption.Key == "" {
```

Comment:
We actually can't change this! The `Key` was the old field and `EncryptedKey` is the current one.

@gioelecerati found out you ended up never merging this: https://github.com/livepeer/go-api-client/pull/41
Probably only worked cause we released `task-runner` with a ref to the PR branch.

Reply:
@gioelecerati I've just merged your PR: https://github.com/livepeer/go-api-client/pull/41 :)

Reply:
Reverted EncryptedKey changes in b00095719f95be725960650b9aba996feb8fa425